### PR TITLE
steves update

### PR DESCRIPTION
**Pull request by stnguyenn**

### CODE REVIEW

**Comment by stnguyenn** [comment]
Khamin , please merge my update.

**Comment by KhaminK** [comment]
Thanks, @stnguyenn .

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# learn-code-review
\ No newline at end of file
+# learn-code-review
+
+## New section
```
Comment (KhaminK) [inline README.md:3]: why are we adding new section?

Comment (stnguyenn) [reply README.md:3]: To show example of pull request.

Comment (KhaminK) [inline README.md:3]: lower case here

**Pull request merged**